app/views/shared/forms/_identify_types_of_outgoings_form.html.erb

```diff
@@ -6,10 +6,8 @@
     ) do |form| %>
 
   <%= govuk_form_group show_error_if: @legal_aid_application.errors.present? do %>
-    <%= govuk_fieldset_header do %>
-      <h1 class="govuk-fieldset__heading govuk-!-padding-bottom-7"><%= page_title %></h1>
-      <h2 class="govuk-heading-m"><%= t('.sub_heading') %></h2>
-    <% end %>
+    <%= govuk_fieldset_header page_title %>
+    <h2 class="govuk-heading-m"><%= t('.sub_heading') %></h2>
```

Comment:
this pattern is used in other pages. And works well in term of spacing. No need to add extra padding class.